Makefile

```diff
@@ -158,22 +159,20 @@ else
 	@echo "scripts/git_hooks/pre-commit.sh file does not exist"
 endif
 
-## Execute git hooks
-hookexec:
-ifneq ("$(wildcard scripts/git_hooks/pre-commit.sh)","")
-	@echo "Executing git hooks"
-	@/bin/sh ./scripts/git_hooks/pre-commit.sh
+## Validate langcode of base config files
+clang:
+ifneq ("$(wildcard scripts/makefile/baseconfig-langcode.sh)","")
+	@echo "Base config langcode validation..."
+	@/bin/sh ./scripts/makefile/baseconfig-langcode.sh
 else
-	@echo "scripts/git_hooks/pre-commit.sh file does not exist"
+	@echo "scripts/makefile/baseconfig-langcode.sh file does not exist"
 endif
 
 ## Inspect configuration
 cinsp:
 	@echo "Config schema validation..."
-	@$(call php, drush -y en config_inspector)
-	@$(eval SCHEMA_ERRORS = $(shell docker-compose exec -T --user $(CUID):$(CGID) php drush inspect_config --only-error))
-	@$(call php, drush -y pmu config_inspector)
-	@if [ ! -z "$(SCHEMA_ERRORS)" ]; then echo "Error(s) in config schemas"; exit 1; fi
+	@$(call php, drush config:inspect --only-error)
 
 ## Full inspection
-insp: | cinsp phpcs
+insp: | phpcs clang cinsp
```

Comment:
I think we could also include `composer validate --strict` here.

Reply:
Agreed ! Let's add it to another PR though (to keep PRs as small and move fast)